test/functionalTest/cases/0000_cli/bool_option_with_value.test

```diff
@@ -85,5 +85,6 @@ Usage: contextBroker  [option '-U' (extended usage)]
                       [option '-strictNgsiv1Ids' (additional checks for id fields in the NGSIv1 API)]
                       [option '-disableCustomNotifications' (disable NGSIv2 custom notifications)]
                       [option '-logForHumans' (human readible log to screen)]
+                      [option '-metrics' (turn off the 'metrics' feature - to gain efficiency)]
```

Comment:
The part of talking about `-disableMetrics` in perf_tunning.md has not been done... I'd try to find the best place to put it.